### ISSUE TITLE
Teleport command produced severe error

### ISSUE THREAD

**Issue by SlimeDog** [SkriptLang/Skript#3521]
### Description
Player session devolved into weird state -- no commands, for example. Player was not teleported to indicated location in a different world, which location exists.

### Steps to Reproduce
With skript below active, allowed player to sustain repeated damage from a zombie attack.

### Expected Behavior
Tick down health from attack (it does), then teleport player to indicated location (produces error).

### Skript
```
on damage of a player:
       damage cause is not fall
       health of the victim is less than 2
       cancel event
       teleport the victim to the spawn location of "world_void"
```

### Errors / Screenshots
```
[10:51:16] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[10:51:16] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[10:51:16] [Server thread/ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[10:51:16] [Server thread/ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Stack trace:
[10:51:16] [Server thread/ERROR]: #!#! java.lang.IllegalStateException: Removing entity while ticking!
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.removeEntity(WorldServer.java:1123)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.removePlayer(WorldServer.java:1142)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PlayerList.moveToWorld(PlayerList.java:606)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer.teleport(CraftPlayer.java:695)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.entity.CraftEntity.teleport(CraftEntity.java:470)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffTeleport.execute(EffTeleport.java:83)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:51)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:60)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:153)
[10:51:16] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:111)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.callEvent(CraftEventFactory.java:251)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:966)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:853)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:809)
[10:51:16] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:998)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.damageEntity0(EntityLiving.java:1755)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityHuman.damageEntity0(EntityHuman.java:828)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.damageEntity(EntityLiving.java:1139)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityHuman.damageEntity(EntityHuman.java:745)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityPlayer.damageEntity(EntityPlayer.java:719)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.attackEntity(EntityInsentient.java:1482)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.attackEntity(EntityZombie.java:281)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalMeleeAttack.a(SourceFile:142)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalMeleeAttack.e(SourceFile:134)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalZombieAttack.e(SourceFile:28)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalWrapped.e(SourceFile:55)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalSelector.doTick(SourceFile:88)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.doTick(EntityInsentient.java:729)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.movementTick(EntityLiving.java:2669)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.movementTick(EntityInsentient.java:530)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityMonster.movementTick(SourceFile:43)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.movementTick(EntityZombie.java:201)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.tick(EntityLiving.java:2398)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.tick(EntityInsentient.java:293)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.tick(EntityZombie.java:171)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.entityJoinedWorld(WorldServer.java:663)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.World.a(World.java:718)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.doTick(WorldServer.java:398)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1104)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:355)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1007)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:846)
[10:51:16] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:164)
[10:51:16] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:834)
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Version Information:
[10:51:16] [Server thread/ERROR]: #!#!   Skript: 2.5.1
[10:51:16] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[10:51:16] [Server thread/ERROR]: #!#!     Date: 2020-10-22
[10:51:16] [Server thread/ERROR]: #!#!   Bukkit: 1.16.4-R0.1-SNAPSHOT
[10:51:16] [Server thread/ERROR]: #!#!   Minecraft: 1.16.4
[10:51:16] [Server thread/ERROR]: #!#!   Java: 11.0.9 (Java HotSpot(TM) 64-Bit Server VM 11.0.9+7-LTS)
[10:51:16] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-1127.19.1.el7.x86_64
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Server platform: Spigot
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Current node: null
[10:51:16] [Server thread/ERROR]: #!#! Current item: teleport the attacked entity to [[direction:at]] spawn of [[world:world_void]]
[10:51:16] [Server thread/ERROR]: #!#! Current trigger: damage of a player (damage of [[entitydata:player]]) (test.sk, line 51)
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Thread: Server thread
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! Language: english
[10:51:16] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[10:51:16] [Server thread/ERROR]: #!#!
[10:51:16] [Server thread/ERROR]: #!#! End of Error.
```

### Server Information
* **Server version/platform:**
```
>version
[11:53:05] [Server thread/INFO]: This server is running CraftBukkit version git-Spigot-37d799b-3eb7236 (MC: 1.16.4) (Implementing API version 1.16.4-R0.1-SNAPSHOT)
[11:53:05] [Server thread/INFO]: You are running the latest version
```
* **Skript version:**
```
>version Skript
[11:53:13] [Server thread/INFO]: Skript version 2.5.1
[11:53:13] [Server thread/INFO]: Customize Minecraft's mechanics with simple scripts written in plain English sentences.
[11:53:13] [Server thread/INFO]: Website: https://skriptlang.github.io/Skript
[11:53:13] [Server thread/INFO]: Authors: Njol, Mirreski, bensku, TheBentoBox, FranKusmiruk, ShaneBeee, Blueyescat, JRoy, Nicofisi, APickledWalrus, xXAndrew28Xx, Syst3ms, TheLimeGlass, Pikachu920, Sashie, Wealthyturtle, OfficialDonut and eyesniper2
```

**Comment by DeltaRays** [comment]
I can't replicate the issue using Skript 2.5.1 and your version of skript after having changed the world to world_nether. Maybe the world you're trying to teleport to isn't loaded?

**Comment by SlimeDog** [comment]
I will check
```
if world("world_void") is loaded
```
when I get back to my computer. Is there a command to ensure that it is loaded, or just a `forceload` on the spawn chunk?

**Comment by SlimeDog** [comment]
world_void spawn chunk is (and surrounding chunks are) loaded by `forceload`. Updated skript reads
```
on damage of a player:
        damage cause is not fall
        health of the victim is less than 2
        world("world_void") is loaded
        broadcast "world_void is loaded"
        teleport the victim to the spawn location of "world_void"
```
Skript reports that `world_void is loaded`, but teleportation does not occur. Error logged
```
[07:55:38] [Server thread/INFO]: world_void is loaded
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! [Skript] Severe Error:
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Something went horribly wrong with Skript.
[07:55:38] [Server thread/ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[07:55:38] [Server thread/ERROR]: #!#! You should report it at https://github.com/SkriptLang/Skript/issues. Please copy paste this report there (or use paste service).
[07:55:38] [Server thread/ERROR]: #!#! This ensures that your issue is noticed and will be fixed as soon as possible.
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Stack trace:
[07:55:38] [Server thread/ERROR]: #!#! java.lang.IllegalStateException: Removing entity while ticking!
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.removeEntity(WorldServer.java:1123)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.removePlayer(WorldServer.java:1142)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PlayerList.moveToWorld(PlayerList.java:606)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.entity.CraftPlayer.teleport(CraftPlayer.java:695)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.entity.CraftEntity.teleport(CraftEntity.java:470)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.effects.EffTeleport.execute(EffTeleport.java:83)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Effect.run(Effect.java:51)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:60)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:88)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler.check(SkriptEventHandler.java:153)
[07:55:38] [Server thread/ERROR]: #!#!     at ch.njol.skript.SkriptEventHandler$1.execute(SkriptEventHandler.java:111)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.callEvent(CraftEventFactory.java:251)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.callEntityDamageEvent(CraftEventFactory.java:966)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:853)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleEntityDamageEvent(CraftEventFactory.java:809)
[07:55:38] [Server thread/ERROR]: #!#!     at org.bukkit.craftbukkit.v1_16_R3.event.CraftEventFactory.handleLivingEntityDamageEvent(CraftEventFactory.java:998)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.damageEntity0(EntityLiving.java:1755)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityHuman.damageEntity0(EntityHuman.java:828)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.damageEntity(EntityLiving.java:1139)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityHuman.damageEntity(EntityHuman.java:745)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityPlayer.damageEntity(EntityPlayer.java:719)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.attackEntity(EntityInsentient.java:1482)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.attackEntity(EntityZombie.java:281)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalMeleeAttack.a(SourceFile:142)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalMeleeAttack.e(SourceFile:134)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalZombieAttack.e(SourceFile:28)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalWrapped.e(SourceFile:55)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:177)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:497)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.PathfinderGoalSelector.doTick(SourceFile:88)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.doTick(EntityInsentient.java:729)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.movementTick(EntityLiving.java:2669)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.movementTick(EntityInsentient.java:530)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityMonster.movementTick(SourceFile:43)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.movementTick(EntityZombie.java:201)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityLiving.tick(EntityLiving.java:2398)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityInsentient.tick(EntityInsentient.java:293)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.EntityZombie.tick(EntityZombie.java:171)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.entityJoinedWorld(WorldServer.java:663)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.World.a(World.java:718)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.WorldServer.doTick(WorldServer.java:398)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.b(MinecraftServer.java:1104)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.DedicatedServer.b(DedicatedServer.java:355)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.a(MinecraftServer.java:1007)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:846)
[07:55:38] [Server thread/ERROR]: #!#!     at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$0(MinecraftServer.java:164)
[07:55:38] [Server thread/ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:834)
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Version Information:
[07:55:38] [Server thread/ERROR]: #!#!   Skript: 2.5.1
[07:55:38] [Server thread/ERROR]: #!#!     Flavor: skriptlang-github
[07:55:38] [Server thread/ERROR]: #!#!     Date: 2020-10-22
[07:55:38] [Server thread/ERROR]: #!#!   Bukkit: 1.16.4-R0.1-SNAPSHOT
[07:55:38] [Server thread/ERROR]: #!#!   Minecraft: 1.16.4
[07:55:38] [Server thread/ERROR]: #!#!   Java: 11.0.9 (Java HotSpot(TM) 64-Bit Server VM 11.0.9+7-LTS)
[07:55:38] [Server thread/ERROR]: #!#!   OS: Linux amd64 3.10.0-1127.19.1.el7.x86_64
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Server platform: Spigot
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Current node: null
[07:55:38] [Server thread/ERROR]: #!#! Current item: teleport the attacked entity to [[direction:at]] spawn of [[world:world_void]]
[07:55:38] [Server thread/ERROR]: #!#! Current trigger: damage of a player (damage of [[entitydata:player]]) (test.sk, line 51)
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Thread: Server thread
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! Language: english
[07:55:38] [Server thread/ERROR]: #!#! Link parse mode: DISABLED
[07:55:38] [Server thread/ERROR]: #!#!
[07:55:38] [Server thread/ERROR]: #!#! End of Error.
```
What else might I try?

**Comment by SlimeDog** [comment]
I think the issue is that CombatLogX does not permit the player to teleport while in combat, which timer has not yet expired.

**Comment by SlimeDog** [comment]
The way around this CLX retraction is by command. I would like to execute
```
execute op command "clx untag %victim%"
```
which unfortunately generates a parsing error
```
>sk reload test
[18:05:56] [Server thread/INFO]: [Skript] Reloading test.sk...
[18:05:56] [Server thread/ERROR]: Use 'attacker' and/or 'victim' in damage events (test.sk, line 71: execute op command "clx untag %victim%"')
[18:05:57] [Server thread/INFO]: [Skript] Encountered 1 error while reloading test.sk!
```
What am I doing wrong?

**Comment by DeltaRays** [comment]
you should use ‘execute console command’

**Comment by SlimeDog** [comment]
Doh! Thanks.
